UITests/TransferMethod/AddTransferMethodBankAccountBusinessTests.swift

```diff
@@ -87,4 +87,24 @@ class AddTransferMethodBankAccountBusinessTests: BaseTests {
 
         addTransferMethod.clickCreateTransferMethodButton()
     }
+
+    func testAddTransferMethod_displayBusinessErrorWithDuplicateData() {
```

Comment:
This class is for Bank Account and not PayPal so you can not add PayPal test case in this class. Also we do have existing testAddTransferMethod_createBankCardDuplicateBankCard() and testAddTransferMethod_createPaypalAccountDuplicateAccount(). So enhance those methods to add following
`app.alerts["Error"].buttons["OK"].tap()
        XCTAssertTrue(app.navigationBars["Bank Account"].exists)`

Reply:
Added the alert in testAddTransferMethod_createPaypalAccountDuplicateAccount. For testAddTransferMethod_createBankCardDuplicateBankCard  duplicate message are shown as inline error message.So the alert is not added.